README.md

```diff
@@ -69,26 +70,61 @@ to some port of your choice e.g.
 "http": [1234, "auto"]
 ```
 
+If you installed OpenSSL in such a way that running `openssl` at a
+command line doesn't work, you also need to adjust the path to the
+OpenSSL binary. This can be done by adding a section to `config.json`
+like:
+
+```
+"ssl": {"openssl": {"binary": "/path/to/openssl"}}
+```
+
 Running Tests Automatically
 ---------------------------
+You will need to install virtualenv before running the cmd line for tests.
+The easiest way to install virtualenv is to add the python scripts directory
+(typically C:\Python27\Scripts) to your `%Path%`
+[Environment Variable](http://www.computerhope.com/issues/ch000549.htm),
+then running easy_install pip
+then running pip install virtualenv
 
-Tests can be run automatically in a browser using the `run` command of
+You will also need to put the appropriate WebDriver executables on the
```

Comment:
This isn't true, except for EdgeDriver because `wpt run` can download the binaries. In the EdgeDriver case it provides a more helpful error at runtime than the text here. My preferred way of fixing this would be a patch to `wpt run` to automatically download EdgeDriver rather than more documentation.